src/file.cpp

```diff
@@ -49,6 +50,15 @@ inline static std::string wstring_to_string(const std::wstring &wstr)
 # endif
 #endif
 
+#ifdef _WASM
+inline static std::string get_cwd_from_env()
+{
+  char* value = getenv("PWD");
+  if (!value) return "(unreachable)";
```

Comment:
This seems to obfuscate the error. Wouldn't it be better to
throw an error or return at least something usable, e.g. "./"?

Reply:
I took `(unreachable)` from [getcwd() man](http://man7.org/linux/man-pages/man3/getcwd.3.html). The libsass doesn't not like "./" as CWD path, so I opted to something "visible".

Shall I use "/" ?

Reply:
I see that all other "cwd" code-paths just throw:
`Exception::OperationError("cwd gone missing")`
Maybe that would be safer since AFAIK libsass
would not know how to handle this anyway.

Reply:
In most of use cases cwd will not be present for the wasm target. Making it throw will be a barrier in usage of the library when user fails to provide an environment variable since libsass initializes the `CWD`. In most of the wasm cases absolute files paths can be provided, making the CWD non-used.

Reply:
I see, in that case I don't know what is actually the best solution.
It might make sense to default it to "/" then, since if I understood
you right, wasm doesn't really have a concept of cwd ...

Reply:
So if I understood it right, wasm doesn't have a concept of cwd (yet?).
Therefore users can define it by defining the PWD environment variable,
and in case this env variable is absent it defaults to the root directory!?

Reply:
That's correct.

Though I think the chances that PWD will be specified is minimal, and not fully work due to the logic in the sassc connected with the CWD (non)usage with output file name.

Reply:
Let's see how this plays out. I guess this might be an area which also needs
clarification on the wasm side. Maybe libsass is a good example for this, as
see use cases either on the web and on the command line. I guess wasm
needs to deal with this at some point but might not be there yet :)